tests/metrics/test_LPIPS.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+import torch
+from skimage import data
+from torchmetrics.image.lpip import LearnedPerceptualImagePatchSimilarity
+
+from meddlr.metrics.LPIPS import LPIPS
+
+
+def test_LPIPS_torchmetrics_reproducibility():
+    """Test reproducibility between SSIM implementations in meddlr and torchmetrics."""
+
+    meddlr_metric = LPIPS(mode="rgb", lpips=True)
+
+    preds = torch.randn((5, 3, 256, 256))
+    targets = torch.randn((5, 3, 256, 256))
+
+    targets = targets + 1j * targets
+    preds = preds + 1j * preds
```

Comment:
don't think you'll run into a case of complex-valued RGB images. this should rpobably be a check in the SSFD/LPIPS implementations as well